src/components/Trade/TradeInput.tsx

```diff
@@ -493,18 +493,34 @@ export const TradeInput = () => {
     }
   }, [isBelowMinSellAmount, feesExceedsSellAmount])
 
+  const handleSellAssetClick = useCallback(
+    (asset: Asset) => handleAssetClick(asset, AssetClickAction.Sell),
+    [handleAssetClick],
+  )
+
+  const handleBuyAssetClick = useCallback(
+    (asset: Asset) => handleAssetClick(asset, AssetClickAction.Sell),
```

Comment:
```suggestion
    (asset: Asset) => handleAssetClick(asset, AssetClickAction.Buy),
```

swapper pretty badly broken here